pkg/config/application.go

```diff
@@ -144,16 +144,16 @@ func RemoveAnchorPeer(config *cb.Config, orgName string, anchorPeerToRemove *Anc
 	return fmt.Errorf("could not find anchor peer %s:%d in %s's anchor peer endpoints", anchorPeerToRemove.Host, anchorPeerToRemove.Port, orgName)
 }
 
-// GetAnchorPeer retrieves existing anchor peers from a application organization.
-func GetAnchorPeer(config *cb.Config, orgName string) ([]*AnchorPeer, error) {
+// GetAnchorPeers retrieves existing anchor peers from a application organization.
+func GetAnchorPeers(config *cb.Config, orgName string) ([]*AnchorPeer, error) {
```

Comment:
It's not specific to this function (and I think I've mentioned it before), please consider value objects on the API.

Reply:
Good point I'll push a quick followon commit to this PR that makes some changes to the types.